spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -381,12 +391,12 @@ The MP-JWT specification requires that an MP-JWT implementation reject a bearer
 invalid MP-JWT token if any of the following conditions are not met:
 
 1. The JWT must have a JOSE header that indicates the token was signed using the RS256 algorithm.
-2. The JWT must have an iss claim representing the token issuer that maps to an MP-JWT implementation
-container runtime configured value. Any issuer other than those issuers that have been whitelisted
-by the container configuration must be rejected with an HTTP_UNUATHENTICATED(401) error.
-3. The JWT signer must have a public key that maps to an MP-JWT implementation container runtime
+2. The JWT signer must have a public key that maps to an MP-JWT implementation container runtime
```

Comment:
`JWT signer` is not a good term as we are not signing a token here but verifying it.

JWT Signature verifier?

Or what exactly we do want to tell the people here?  When a key identification (kid then) is encountered which is not expected we need to return 401?

Reply:
@rdebusscher I've just moved the existing lines around there, the whole `interoperability.doc` needs a good refresh but `JWT Signature verifier` is minor but a good start. I believe `kid` is mentioned elsewhere in scope of the JWK format text, this is just a high level requirement that a public key should be available.

Reply:
@rdebusscher I've read that text and it just requires that a signer which signs (with a private key) should have a matching public key that can be mapped by MP JWT server somehow. We cam tweak for sure, but I'd rather have a dedicated issue (Improve/clarify the texts etc) as it can easily take few days :-)